test/cpp_extensions/cudnn_extension.cpp

```diff
@@ -0,0 +1,73 @@
+#include <torch/torch.h>
+#include <cuda.h>
+#include <torch/torch.h>
+#include <ATen/ATen.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/cudnn/Descriptors.h>
+#include <ATen/cudnn/Exceptions.h>
+#include <ATen/cudnn/Handles.h>
+#include <torch/csrc/cuda/cuda_check.h>
+#include <torch/torch.h>
+#include <iostream>
+
+using namespace at;
+using namespace std;
+using namespace at::native;
+const char *cudnn_relu_name = "cudnn_relu";
+
+// Check arguments to cudnn_relu
+int cudnn_relu_check(const Tensor &inputs, const Tensor &outputs) {
+  // Create TensorArgs
+  TensorArg arg_inputs(inputs, "inputs", 0);
+  TensorArg arg_outputs(outputs, "outputs", 1);
+  // Check arguments
+  checkContiguous(cudnn_relu_name, arg_inputs);
+  checkScalarType(cudnn_relu_name, arg_inputs, ScalarType::Float);                      \
+  checkBackend(cudnn_relu_name, arg_inputs.tensor, Backend::CUDA);
+  checkContiguous(cudnn_relu_name, arg_outputs);
+  checkScalarType(cudnn_relu_name, arg_outputs, ScalarType::Float);                      \
```

Comment:
Prefer using `at::kFloat` instead of `at::ScalarType::Float` and `at::kCUDA` instead of `at::Backend::CUDA`